lib/Controller/EndpointController.php

```diff
@@ -195,10 +195,10 @@ public function confirmIdsForUser(string $apiVersion, array $ids): DataResponse
 			return new DataResponse([], Http::STATUS_BAD_REQUEST);
 		}
 
-		$ids = array_unique(array_filter(array_map(
+		$ids = array_values(array_unique(array_filter(array_map(
```

Comment:
This is not what we return, so I don't really get why this is needed (just for understanding it)

Reply:
Yes, but it is cleaner to make the `confirmIdsForUser()` also only accept and return `list<int>`.
Technically not needed to change this, but I did it for completeness.